src/main/java/pl/agh/cs/io/RuleRestriction.java

```diff
@@ -0,0 +1,61 @@
+package pl.agh.cs.io;
+
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+
+public class RuleRestriction {
+
+    public final WindowState state;
+    public final long permittedNumSec;
+
+    private ExceededUsageAction action;
+    private Rule rule;
+
+    private long lastNotification;
+    private long numOfSecBetweenNotifications = 5;
+
+
+    public RuleRestriction(WindowState state, long permittedNumSec, ExceededUsageAction action) {
+        this.permittedNumSec = permittedNumSec;
+        this.state = state;
+        this.lastNotification = 0;
+        this.action = action;
+    }
+
+    public void checkRestriction() {
+        if (rule != null && permittedNumSec > 0) {
+            double usedToday = ActivityTime.getActivityTimeFromList(rule.getTimes(), state);
```

Comment:
Mamy pewność, że to jest tylko dzisiaj?

Reply:
Na razie nie mamy wprowadzonej persystencji, dopiero jak ją wprowadzimy będziemy mogli pobierać dokładny dzień